testdata/test-data-systematic.xml

```diff
@@ -693,6 +691,35 @@
         </color-prop>
     </region>
 
+    <region label="Circle: dysfunctional"
```

Comment:
It is still an open question how circles can be created. Here I just added a draft which is not functional yet. It doesn't affect the test runs, the only issue is that it isn't displayed in the APP (and I suppose that the API cannot handle it)

Reply:
who is deciding that? and how?  
Also, should it really be a circle, or rather an ellipsis? (in which case the radius and center would not work)

Reply:
I meant "draft" in the sense of a try. I think that this functionality is somewhere hidden in the API, but it is  documented nowhere. So I try to find the right syntax. Is there a method how I could find the syntax?

Reply:
In my opinion, if it doesn't work in DSP-APP it should not be in the docs of DSP-TOOLS. Otherwise we confuse users, don't you think so?
For circles, you would find an example in the incunabula test data: https://raw.githubusercontent.com/dasch-swiss/dsp-api/main/test_data/all_data/incunabula-data.ttl (open this page and search for "circle")